### PR TITLE
Feature/flux model v1 2 2 run

### PR DESCRIPTION
**Pull request by dagibbs22**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [x] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?
Previous version used flux model v.1.2.1 (2001-2020 tree cover loss) outputs.

## What is the new behavior?
- Uses flux model v1.2.2 (2001-2021 tree cover loss) outputs.
- Emissions now includes columns for emissions from soil only (not just biomass and soil).
- Change tables include removals by year of tree cover loss.
- Made output fields match the API (annualupdate_minimal package) output fields.
- Used other changes to Geotrellis that Justin had implemented, plus a few little fixes to make it launch correctly from the sbt shell in IntelliJ.
- Reverted to using 200 worker instances (100 in previous year, I believe).

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

N/A

### CODE REVIEW

File: src/test/resources/palm-32-fcd-output/part-00000-50e62995-5e11-45dc-83c0-0c97a8427354-c000.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
...id	location_id	status_code	location_error	tree_cover_loss_total_yearly	tree_cover_loss_primary_forest_yearly	tree_cover_loss_peat_yearly	tree_cover_loss_intact_forest_yearly	tree_cover_loss_protected_areas_yearly	tree_cover_loss_sea_landcover_yearly	tree_cover_loss_idn_landcover_yearly	tree_cover_loss_soy_yearly	tree_cover_loss_idn_legal_yearly	tree_cover_loss_idn_forest_moratorium_yearly	tree_cover_loss_prodes_yearly	tree_cover_loss_prodes_wdpa_yearly	tree_cover_loss_prodes_primary_forest_yearly	tree_cover_loss_brazil_biomes_yearly	tree_cover_extent_total	tree_cover_extent_primary_forest	tree_cover_extent_protected_areas	tree_cover_extent_peat	tree_cover_extent_intact_forest	natural_habitat_primary	natural_habitat_intact_forest	total_area	protected_areas_area	peat_area	brazil_biomes	idn_legal_area	sea_landcover_area	idn_landcover_area	idn_forest_moratorium_area	south_america_presence	legal_amazon_presence	brazil_biomes_presence	cerrado_biome_presence	southeast_asia_presence	indonesia_presence	commodity_value_forest_extent	commodity_value_peat	commodity_value_protected_areas	commodity_threat_deforestation	commodity_threat_peat	commodity_threat_protected_areas	commodity_threat_fires
```
Comment (jterry64) [inline src/test/resources/palm-32-fcd-output/part-00000-50e62995-5e11-45dc-83c0-0c97a8427354-c000.csv:1]: why did you add this file?

**Review by jterry64 — CHANGES_REQUESTED** [review]

File: src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossTile.scala [hunk 2]
```diff
@@ -21,8 +21,8 @@ case class TreeLossTile(loss: TreeCoverLoss#ITile,
                         grossCumulAbovegroundRemovalsCo2: GrossCumulAbovegroundRemovalsCo2#OptionalFTile,
                         grossCumulBelowgroundRemovalsCo2: GrossCumulBelowgroundRemovalsCo2#OptionalFTile,
                         netFluxCo2: NetFluxCo2e#OptionalFTile,
-                        grossEmissionsCo2eNonCo2: GrossEmissionsNonCo2Co2e#OptionalFTile,
-                        grossEmissionsCo2eCo2Only: GrossEmissionsCo2OnlyCo2e#OptionalFTile,
+                        grossEmissionsCo2eNonCo2: GrossEmissionsNonCo2Co2eBiomassSoil#OptionalFTile,
```
Comment (jterry64) [inline src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossTile.scala:24]: Why is this biomass soil now?

File: src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossGridSources.scala [hunk 3]
```diff
@@ -24,8 +24,8 @@ case class TreeLossGridSources(gridTile: GridTile, kwargs: Map[String, Any]) ext
 
   val grossCumulAbovegroundRemovalsCo2: GrossCumulAbovegroundRemovalsCo2 = GrossCumulAbovegroundRemovalsCo2(gridTile, kwargs = kwargs)
   val grossCumulBelowgroundRemovalsCo2: GrossCumulBelowgroundRemovalsCo2 = GrossCumulBelowgroundRemovalsCo2(gridTile, kwargs = kwargs)
-  val grossEmissionsCo2eNonCo2: GrossEmissionsNonCo2Co2e = GrossEmissionsNonCo2Co2e(gridTile, kwargs = kwargs)
-  val grossEmissionsCo2eCo2Only: GrossEmissionsCo2OnlyCo2e = GrossEmissionsCo2OnlyCo2e(gridTile, kwargs = kwargs)
+  val grossEmissionsCo2eNonCo2: GrossEmissionsNonCo2Co2eBiomassSoil = GrossEmissionsNonCo2Co2eBiomassSoil(gridTile, kwargs = kwargs)
```
Comment (jterry64) [inline src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossGridSources.scala:27]: Why is this biomass soil now?

**Review by jterry64 — CHANGES_REQUESTED** [review]

File: src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossTile.scala (hunk 2 above)
Comment (dagibbs22) [reply src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossTile.scala:24]: Those emissions are called biomasssoil now because I added three new layers that are emissions for soil_only (as opposed to biomass + soil emissions). So I added biomasssoil to distinguish them from the soil_only emissions. In the carbonflux package, I need to distinguish between biomasssoil and soil_only but treecoverloss only uses biomasssoil. Is it okay to leave this change for clarity, though?

File: src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossGridSources.scala (hunk 3 above)
Comment (dagibbs22) [reply src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossGridSources.scala:27]: Those emissions are called biomasssoil now because I added three new layers that are emissions for soil_only (as opposed to biomass + soil emissions). So I added biomasssoil to distinguish them from the soil_only emissions. In the carbonflux package, I need to distinguish between biomasssoil and soil_only but treecoverloss only uses biomasssoil. Is it okay to leave this change for clarity, though?

**Review by jterry64 — APPROVED** [review]
